client/state/account-recovery/actions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import wpcom from 'lib/wp';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ACCOUNT_RECOVERY_FETCH,
+	ACCOUNT_RECOVERY_FETCH_SUCCESS,
+	ACCOUNT_RECOVERY_FETCH_FAILED,
+} from 'state/action-types';
+
+export const accountRecoveryFetchSuccess = ( accountRecoverySettings ) => {
+	return {
+		type: ACCOUNT_RECOVERY_FETCH_SUCCESS,
+		accountRecoverySettings,
```

Comment:
It might be worth breaking out the settings here to be explicit instead of passing the whole object, particularly since we transform them to camelCase later:

```js
export const accountRecoveryFetchSuccess = ( { email, email_validated, phone, phone_validated } ) => ( {
  type: ACCOUNT_RECOVERY_FETCH_SUCCESS,
  email,
  email_validated,
  phone,
  phone_validated,
} );
```

Or,

```js
export const accountRecoveryFetchSuccess = ( accountRecoverySettings ) => ( {
  type: ACCOUNT_RECOVERY_FETCH_SUCCESS,
  ...accountRecoverySettings,
} );
```